client/.eslintcache

```diff
@@ -1 +1 @@
-[{"/home/lina/Desktop/G9/calorie-tracker/client/src/index.js":"1","/home/lina/Desktop/G9/calorie-tracker/client/src/App.js":"2"},{"size":381,"mtime":1612122842050,"results":"3","hashOfConfig":"4"},{"size":148,"mtime":1612122647845,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"8"},"1c9swl7",{"filePath":"9","messages":"10","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/home/lina/Desktop/G9/calorie-tracker/client/src/index.js",[],["11","12"],"/home/lina/Desktop/G9/calorie-tracker/client/src/App.js",[],{"ruleId":"13","replacedBy":"14"},{"ruleId":"15","replacedBy":"16"},"no-native-reassign",["17"],"no-negated-in-lhs",["18"],"no-global-assign","no-unsafe-negation"]
\ No newline at end of file
+[{"/home/khamis/Desktop/geeks-G9/calorie-tracker/client/src/index.js":"1","/home/khamis/Desktop/geeks-G9/calorie-tracker/client/src/App.js":"2","/home/khamis/Desktop/geeks-G9/calorie-tracker/client/src/components/Chart.js":"3"},{"size":168,"mtime":1612263041740,"results":"4","hashOfConfig":"5"},{"size":203,"mtime":1612263041736,"results":"6","hashOfConfig":"5"},{"size":844,"mtime":1612264548575,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"serryl",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},{"filePath":"13","messages":"14","errorCount":1,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"/home/khamis/Desktop/geeks-G9/calorie-tracker/client/src/index.js",[],["15","16"],"/home/khamis/Desktop/geeks-G9/calorie-tracker/client/src/App.js",[],"/home/khamis/Desktop/geeks-G9/calorie-tracker/client/src/components/Chart.js",["17","18"],{"ruleId":"19","replacedBy":"20"},{"ruleId":"21","replacedBy":"22"},{"ruleId":"23","severity":1,"message":"24","line":2,"column":10,"nodeType":"25","messageId":"26","endLine":2,"endColumn":18},{"ruleId":"27","severity":2,"message":"28","line":7,"column":15,"nodeType":"25","messageId":"29","endLine":7,"endColumn":20},"no-native-reassign",["30"],"no-negated-in-lhs",["31"],"no-unused-vars","'Doughnut' is defined but never used.","Identifier","unusedVar","no-undef","'props' is not defined.","undef","no-global-assign","no-unsafe-negation"]
```

Comment:
please remove the eslintcache using git rm client/.eslintcache --cached and then add it to .gitignore